Student/StudentUITests/Submissions/SubmissionDetails/SubmissionDetailsTests.swift

```diff
@@ -209,6 +209,8 @@ class SubmissionDetailsTests: StudentUITestCase {
         ]))
         mockDownload(downloadURL, data: url)
 
+        // There's a uuid in a request that has no way to be mocked currently
```

Comment:
We do have a wrapper, `UUID.swift`, that supports mocking, in case you didn't already know. We will need to update the test helper to mock it though.

https://github.com/instructure/canvas-ios/blob/9aa05ce01db37aac8a57570c144efcdbba8c4509/Core/Core/AppEnvironment/UUID.swift#L25-L27